jobs/e2e_node/image-config-serial-cpu-manager.yaml

```diff
@@ -3,7 +3,7 @@
 # `gcloud compute --project <to-project> images create <image-name> --source-disk=<image-name>`
 images:
   ubuntu:
-    image: ubuntu-gke-1604-xenial-v20170420-1 # docker 1.12.6
+    image: ubuntu-gke-1804-d1703-0-v20181113 # docker 17.03
```

Comment:
Could we use ubuntu-gke-1804-d1703-0-v20190917 ?

Reply:
I guess we can. I have not tested that specific one tho. `v20181113` is used for all the other node_e2e tests (
https://github.com/kubernetes/test-infra/search?q=ubuntu-gke-1804-d1703-0-v20181113&unscoped_q=ubuntu-gke-1804-d1703-0-v20181113), so I think it's best to bump all of them at the same time. Or what do you think @cheftako ?

Reply:
/cc vpickard

Reply:
Maybe use the latest `ubuntu-gke-1804-d1903-0-*` which has Docker 19.03?